src/AccessControl/SecurityManagement.py

```diff
@@ -42,7 +42,7 @@ def setSecurityManager(manager):
 # AccessControl.Implementation inserts SecurityManager.
 
 try:
-    from thread import get_ident
+    from AccessControl._compat import get_ident
```

Comment:
The `try: ... except:` should go away, as `_compat` makes it work unconditionally.

Reply:
Done.